lib/models/services-schema.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
The name of this file should reflect that it is the definition of the `Instance` model (more than just a schema). You can see how we name our model definition files in [HUD](https://github.com/inetCatapult/hud/tree/master/lib/models).